Model/Site.php

```diff
@@ -151,14 +151,28 @@ public function isEnabled()
      */
     public function getUrl()
     {
+        if ($this->isSecureRequest()) {
+            $protcol = 'https';
+        } else {
+            $protcol = 'http';
+        }
+
         if ($this->isLocalhost()) {
             return $this->getRelativePath();
         }
 
-        return sprintf('http://%s%s', $this->getHost(), $this->getRelativePath());
+        return sprintf($protcol.'://%s%s', $this->getHost(), $this->getRelativePath());
```

Comment:
See http://stackoverflow.com/questions/9646407/two-forward-slashes-in-a-url-src-href-attribute